rustls/src/crypto/hmac.rs

```diff
@@ -45,11 +52,45 @@ impl Tag {
 
 impl Drop for Tag {
     fn drop(&mut self) {
-        self.buf.zeroize();
+        self.0.buf.zeroize();
     }
 }
 
 impl AsRef<[u8]> for Tag {
+    fn as_ref(&self) -> &[u8] {
+        self.0.as_ref()
+    }
+}
+
+/// A HMAC tag, stored as a value, that is non-secret.
```

Comment:
bike-shedding: What about putting the most relevant part in the headline up-front like:

> A non-secret HMAC tag, stored as a value.